### PR TITLE
Feature: added randomised death messages for LostDialog

### PR DESCRIPTION
**Pull request by TheBeville**
For #27

### CODE REVIEW

File: lib/game_page.dart [hunk 1]
```diff
@@ -46,6 +48,19 @@ class _GamePageState extends State<GamePage> {
...
+  final List<String> deathMessages = [
+    'YOU DIED',
+    'wasted',
+    'We\'ll get \'em next time!',
+    'Ope, you lose',
+    'We regret to inform you that you have been unalived',
+    'Character status: deceased',
+  ];
+
+  int pickDeathMessage() {
```
Comment (alexobviously) [inline lib/game_page.dart:60]: you could just do `String pickDeathMessage() => deathMessages[Random().nextInt(deathMessages.length)];`

**Review by alexobviously — APPROVED** [review]

**Comment by alexobviously** [comment]
approved but look at comment

File: lib/game_page.dart [hunk 2]
```diff
@@ -46,6 +48,19 @@ class _GamePageState extends State<GamePage> {
     settings().setRayDensity(density);
   }
 
+  final List<String> deathMessages = [
```
[the remark this answers is not in the document]
Comment (TheBeville) [reply lib/game_page.dart:51]: Unsure how I missed that

File: lib/game_page.dart (hunk 1 above)
Comment (TheBeville) [reply lib/game_page.dart:60]: Changed to your suggestion.

**Pull request merged**